docs/api/4.0.17.md

```diff
@@ -2,16 +2,18 @@
 
 ## Math
 
-### percentile *<a target="_blank" href="https://wso2.github.io/siddhi/documentation/siddhi-4.0/#aggregate-function">(Aggregate Function)</a>*
+### percentile *<a target="_blank" href="https://wso2.github.io/siddhi/documentation/siddhi-4
+.0/#aggregate-function">(Aggregate Function)</a>*
 
-<p style="word-wrap: break-word">Returns the pth percentile value of the arg values.</p>
+<p style="word-wrap: break-word">Returns the pth percentile value of the given argument****_``_**** values.</p>
```

Comment:
api/x.x.x.md are auto generated files so the fixes will be lost when new version is built. 
Hence please fix the same in source files at  https://github.com/wso2-extensions/siddhi-execution-math/tree/master/component/src/main/java/org/wso2/extension/siddhi/execution/mat